test/cloud_traces

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
Before diving into the review. Could you please provide an example output here for testing?

Reply:
Sure, I have added tests and made a minor change to an error. Please let me know any further changes.